bundles/org.smarthomej.binding.androiddebugbridge/src/main/java/org/smarthomej/binding/androiddebugbridge/internal/AndroidDebugBridgeHandler.java

```diff
@@ -139,14 +143,18 @@ private void handleCommandInternal(ChannelUID channelUID, Command command)
                 break;
             case AWAKE_STATE_CHANNEL:
                 if (command instanceof RefreshType) {
-                    boolean awakeState = adbConnection.isAwake();
-                    updateState(channelUID, OnOffType.from(awakeState));
+                    Boolean awakeState = adbConnection.isAwake();
+                    if (awakeState != null) {
+                        updateState(channelUID, OnOffType.from(awakeState));
+                    }
```

Comment:
fixed, but do not understand why to make it more complicated with optional instead to use null as a status

Reply:
Using `Optional`prevents NPE. If you return `null` you need to check if the return-value is null (and this could be forgotten). Another option would be to return the primitive `boolean` and thrwo a checked exception (e.g. `ValueNotFoundException` or something names similar) and catch that. But usually the `Optional` is easier and requires less code.